pkg/kubelet/util/util_windows.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"syscall"
 	"time"
+
+	"github.com/Microsoft/hcsshim/hcn"
```

Comment:
could you rebase? Not sure why but this seems to be referencing the old library. We recently removed this library and replaced with "github.com/Microsoft/hnslib/hcn"

https://github.com/kubernetes/kubernetes/pull/128045/files

Reply:
replaced.